src/common/accelcmn.cpp

```diff
@@ -38,51 +38,50 @@ wxAcceleratorTable wxNullAcceleratorTable;
 // wxAcceleratorEntry implementation
 // ============================================================================
 
-wxGCC_WARNING_SUPPRESS(missing-field-initializers)
-
 static const struct wxKeyName
 {
     wxKeyCode code;
     const char *name;
     const char *display_name;
 } wxKeyNames[] =
 {
-    { WXK_DELETE,           /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Delete") },
-    { WXK_DELETE,           /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Del") },
+    { WXK_DELETE,           /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Delete"),        /*Display is the same as key name*/ 0 },
```

Comment:
I'm not really sure we gain anything from doing this... Is having a lot of redundant zeroes/comments in the array definition really better than disabling the (well-meaning but pretty useless in this particular case) warning?

Reply:
I had done this because I was always leery of trusting compilers to initialize memory properly (something I picked up from C I guess), so I figured since these values are being used in the checks we should ensure they are 0. I had tried setting them to empty string (just by putting "" into the column) but then that does create the pointer so the checks are no longer useful. So I wasn't sure how the compiler might try to initialize them (if at all) and wanted to force their value to 0.

If you generally see that compilers initialize them properly, I can revert this part and leave them empty. My general rule of thumb is to always initialize though.

Reply:
Yes, I'm pretty sure all the fields are initialized to 0 by the C++ compiler (I know that the rules differ here between C++ and C, but I don't remember what the situation is in C to be honest). Not only is this required by the standard, but I've also never seen this not to happen in practice.

Reply:
Ok, since the initialization is in the standard I will remove the 0 initialization.